content/en/docs/collector/deployment-methods.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Deployment Methods
+weight: 2
+---
+
+The OpenTelemetry Collector consists of a single binary and two primary deployment methods:
+
+- **Agent:** A Collector instance running with the application or on the same
+  host as the application (e.g. binary, sidecar, or daemonset).
+- **Gateway:** One or more Collector instances running as a standalone service
+  (e.g. container or deployment) typically per cluster, data center or region.
+
+### Agent
```

Comment:
The headings in this page should be a level 2 (here and below).

Reply:
fixed at 0941baf